iac-extensions/kubernetes/src/test/resources/checks/ChecksGoTemplate/helm/templates/capabilities-pod.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+spec:
+  containers:
+    - image: {{ .Values.image.name }}
+      command: ["sleep", "6000"]
+      name: test-container
+      securityContext:
+        capabilities:
+          add: {{ .Values.capabilities | toYaml | nindent 10 }} {{- /* Noncompliant {{Check hardcoded issue from Go AST}} */}}
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+spec:
+  containers:
+    - image: {{ .Values.image.name }}
+      command: ["sleep", "6000"]
+      name: test-container
+      securityContext:
+        capabilities:
+          {{- /* Note: noncompliant@+1 on a separate line doesn't work because of the line shifting logic */}}
+          add: {{ .Values.capabilities | toYaml | nindent 10 }}
+#         {{- /* Noncompliant {{Check hardcoded issue from Go AST}} */}}
+#         {{- /* ^[sc=19;ec=38] */}}
```

Comment:
This interacts a bit weirdly with line shifting mechanism, that's why it's not `Noncompliant@-1`. If a noncompliant comment is on the same line as some YAML code, then the line is calculated corrently